### DISCUSSION TITLE
TODOMVC in Lona?

### DISCUSSION THREAD

**Discussion by metaperl** [lona-web-org/lona#538] [General]
Hello, the [Pure Python web framework survey](https://metaperl.github.io/pure-python-web-development) has a [section with TODOMVC examples](https://metaperl.github.io/pure-python-web-development/todomvc.html) ... would you consider adding such an example to the demos section of the Lona docs - https://lona-web.org/1.x/demos/index.html

**Comment by fscherf** [comment]
Hi @metaperl!
I never heard of that project before. It seems to be a benchmark for the DX of JavaScript Frameworks. Does that fit with Lona? Lona is not a JavaScript Framework

**Reply by metaperl** [reply]
What does `DX` stand for?

> Does that fit with Lona? 

The other leading pure python web frameworks [have examples of TODOMVC](https://metaperl.github.io/pure-python-web-development/todomvc.html).

> Lona is not a JavaScript Framework

yes but it should be able to implement TODOMVC as the other leading pure python web frameworks have done.
